weblate_web/static/style-rtl.css

```diff
@@ -4032,71 +4047,81 @@ input.fullwidth {
   }
 }
 
-/* Your Weblate Team - About Us */
```

Comment:
You remove "Your Weblate Team" section styling here:

![image](https://user-images.githubusercontent.com/212189/107944412-59f1f380-6f8e-11eb-8acb-5af80053462d.png)

Reply:
Fixing it!

Reply:
Please check the recent commit.